File: Assignment 3/cpp/a3q2.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){

    int input = 0;
    

    while (input <=0 || input > 10)
    {
        cout << "Enter an integer between 1 and 10 inclusive: ";
        cin >> input;
    }

    for (size_t i = 1; i < input + 1; i++)
    {
        for (size_t j = 1; j < input + 1; j++)
        {
            cout << i*j << "\t";
        }
        cout << endl;
    }
    


}

File: Assignment 3/java/a3q2.java
/*Regan Ross 
(0369345)
COSC 1046
 Dr. Aaron Langille, 
 Champion of the Skittle Toss
 */
 
import java.util.Scanner;
public class Nested
{
	public static void main(String[] args)
	{
		Scanner s = new Scanner(System.in);
		System.out.print("Enter an integer between 1 and 10: ");
		int input = s.nextInt();
		
		if(1<=input && input<= 10)
		{
		
			for(int i = 1; i <= input; i++)
			{
				for(int t = 1; t <= input; t++)
				{
					System.out.print(i*t +"\t");
				}
		
				System.out.println();
			}
		
		}
		
		
		System.out.println("Program Complete.");
	}
	
}
